contracts/v0.2/src/FeedRouter.sol

```diff
@@ -16,15 +16,17 @@ import {IFeedProxy} from "./interfaces/IFeedProxy.sol";
  * can find all supported tokens at https://config.orakl.network.
  */
 contract FeedRouter is Ownable, IFeedRouter {
-    mapping(string => address) public feedProxies;
+    mapping(string => address) public feedToProxies;
```

Comment:
Just curious, is there a specific reason why you thought `feedToProxies` is better than `feedProxies`?

Reply:
No, specific reason. At first I was using different variable names (`feeds` -> `feedNames`) and the name was too similar to `feedProxies` so I renamed it, but now it has no problem so I can move it back.

Reply:
I don't mind :) thank you for the explanation